adafruit_sgp30.py

```diff
@@ -75,6 +75,40 @@ def __init__(self, i2c, address=_SGP30_DEFAULT_I2C_ADDR):
             raise RuntimeError('SGP30 Not detected')
         self.iaq_init()
 
+        self._co2eq = None # pylint: disable=invalid-name
+        self._tvoc = None # pylint: disable=invalid-name
+        self._baseline_tvoc = None # pylint: disable=invalid-name
+        self._baseline_co2eq = None # pylint: disable=invalid-name
+
+
+    @property
+    def tvoc(self): # pylint: disable=invalid-name
```

Comment:
These look like valid names to me. Did you copy the lint part from somewhere else? I don't think they are needed.

Reply:
Yeah, I was copying that from the CCS811 library. I'll remove.